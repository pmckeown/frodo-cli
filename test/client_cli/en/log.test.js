import cp from 'child_process';
import { promisify } from 'util';

const exec = promisify(cp.exec);

test("CLI help interface for 'log' should be expected english", async () => {
  const CMD = 'frodo log --help';
  const { stdout } = await exec(CMD);
  expect(stdout).toMatchSnapshot();
});

test("CLI help interface for 'logs' should be expected english", async () => {
  const CMD = 'frodo logs --help';
  const { stdout } = await exec(CMD);
  expect(stdout).toMatchSnapshot();
});
