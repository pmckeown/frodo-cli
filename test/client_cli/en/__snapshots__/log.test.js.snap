// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI help interface for 'log' should be expected english 1`] = `
"Usage: frodo log|logs [options] [command]

View Identity Cloud logs. If valid tenant admin credentials are specified, a log API key and secret are automatically created for that admin user.

Options:
  -h, --help      Help

Commands:
  fetch           Fetch Identity Cloud logs for a time window.
  help [command]  display help for command
  key             Manage Identity Cloud log API keys.
  list            List available ID Cloud log sources.
  tail            Tail Identity Cloud logs.
"
`;

exports[`CLI help interface for 'logs' should be expected english 1`] = `
"Usage: frodo log|logs [options] [command]

View Identity Cloud logs. If valid tenant admin credentials are specified, a log API key and secret are automatically created for that admin user.

Options:
  -h, --help      Help

Commands:
  fetch           Fetch Identity Cloud logs for a time window.
  help [command]  display help for command
  key             Manage Identity Cloud log API keys.
  list            List available ID Cloud log sources.
  tail            Tail Identity Cloud logs.
"
`;
